### PR TITLE
Fix GhApi owner param

### PR DESCRIPTION
**Pull request by ashwindasr**
For `pyartcd:WARNING No assembly PRs were found with head=thegreyd:art8712` issue

### CODE REVIEW

File: pyartcd/pyartcd/pipelines/build_sync.py [hunk 1]
```diff
@@ -24,6 +24,7 @@
 
 TRACER = trace.get_tracer(__name__)
 GEN_PAYLOAD_ARTIFACTS_OUT_DIR = 'gen-payload-artifacts'
+GITHUB_OWNER = "openshift-eng"
```
Comment (locriandev) [inline pyartcd/pyartcd/pipelines/build_sync.py:27]: Already defined [here](https://github.com/openshift-eng/art-tools/blob/main/pyartcd/pyartcd/pipelines/build_microshift.py#L431). Can we move it to a common place? e.g. https://github.com/openshift-eng/art-tools/blob/main/pyartcd/pyartcd/constants.py

**Review by locriandev — CHANGES_REQUESTED** [review]

File: pyartcd/pyartcd/pipelines/build_sync.py (hunk 1 above)
Comment (ashwindasr) [reply pyartcd/pyartcd/pipelines/build_sync.py]: Thanks! Updated

**Review by locriandev — COMMENTED** [review]
/lgtm

**Pull request merged**